packages/components/src/__overlays__/Tooltip/v3/_docs/Tooltip.spec.stories.tsx

```diff
@@ -33,6 +33,8 @@ export default meta
 
 type Story = StoryObj<typeof meta>
 
+// TODO: update this to use the new button component v3
```

Comment:
Don't forget this

Reply:
Just recapping from something we discussed a while back but will tackle these in the work to migrate all the V1/2 buttons to v3 later :)